pilot/pkg/serviceregistry/kube/controller/controller_test.go

```diff
@@ -2472,7 +2472,7 @@ func TestWorkloadInstanceHandlerMultipleEndpoints(t *testing.T) {
 	// we should have the pod IP and the workload Entry's IP in the endpoints..
 	// the first endpoint should be that of the k8s pod and the second one should be the workload entry
 
-	var gotEndpointIPs []string
+	gotEndpointIPs := make([]string, 0, len(ev.Endpoints))
```

Comment:
nit: I don't mind this but in the future, IMO, readability/simplicity > performance in tests